health/README.md

```diff
@@ -665,6 +665,43 @@ Note that the drops chart does not exist if a network interface has never droppe
 When Netdata detects a dropped packet, it will add the chart and it will automatically attach this
 alarm to it.
 
+### Example 5
+
+Check if user or system dimension is using more than 50% of cpu:
+
+```
+ alarm: dim_template
+    on: system.cpu
+    os: linux
+lookup: average -3s percentage foreach system,user
+ units: %
+ every: 1s
```

Comment:
minimum recommended `every` is 10s, check other exanmples

Reply:
Thank you @ilyam8 , I fixed in both examples.